haikufinder/__init__.py

```diff
@@ -71,8 +71,8 @@ def read_alternates(which):
     syllables = pickle.load(p)
 with open(file('cmudict/custom.dict'), 'r') as p:
     for line in p.readlines():
-    	if not len(line):
-    		continue
+        if not len(line):
```

Comment:
OMG yes thank you for this. These tabs kept the code from running in Python 3.

But while you're in here, I think it's more Pythonic to just do `if not line:` instead of checking against `len(line)`